README.md

```diff
@@ -169,3 +169,5 @@ I'm getting an error about a missing typescript install.
 Error: Cannot find module '/node_modules/dtslint/typescript-installs/3.1/node_modules/typescript`
 ```
 Package lock files such as `yarn.lock` and `package-lock.json` may cause this issue because of our github dependency on `"definitelytyped-header-parser": "github:Microsoft/definitelytyped-header-parser#production"`, which contains the list of typescript versions to install. To fix this, try deleting your lock file and re-installing.

```

Comment:
> Package lock files such as `yarn.lock` and `package-lock.json` may cause this issue because of our github dependency on `"definitelytyped-header-parser": "github:Microsoft/definitelytyped-header-parser#production"`, which contains the list of typescript versions to install. To fix this, try deleting your lock file and re-installing.

Separate topic but can this text be removed since https://github.com/microsoft/dtslint/commit/5e0f2e5de61a944f897a1b1f8066e87b63c7b2ea, or #287?

Reply:
I think you can still run into the problem, but the explanation should just be "Your dependencies may be out of date."

package-lock.json is checked into dtslint but not Definitely Typed, so maybe the solution can just be `npm ci`? But only if you're doing it from this repo and not Definitely Typed.